Gemfile.lock

```diff
@@ -34,7 +38,7 @@ GEM
     crass (1.0.6)
     diff-lcs (1.3)
     erubi (1.9.0)
-    google-protobuf (3.12.2)
+    google-protobuf (3.12.2-universal-darwin)
```

Comment:
seems osx specific, is that intentional?

Reply:
It wasn't intentional. I was thinking the same thing, but it didn't seem to break anything in CI. I'll revert it just in case.